bar-trender-front-end/src/App.js

```diff
@@ -27,6 +27,7 @@ export default class App extends React.Component {
           headers: {
             'Content-Type': 'application/json',
             'token': token,
+            'apiKey':'8dDc431125634ef43cD13c388e6eCf11'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7167715)
```suggestion
            "apiKey":'8dDc431125634ef43cD13c388e6eCf11'
```